### PR TITLE
Add pytables

### PR DESCRIPTION
**Pull request by ocefpaf**
Absorbed #1133 here for now just to check if `pytables` works.

### CODE REVIEW

**Comment by ocefpaf** [comment]
OK. `Travis-CI` seems OK besides the time and `CircleCI` also did fine. Windows seems to be complicated. So I will skip it for now.

**Comment by ocefpaf** [comment]
`CircleCI` will pass once we merge #1133

File: recipes/pytables/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    url: https://github.com/PyTables/PyTables/archive/v.{{ version }}.tar.gz
+    sha256: e422a57f0507d03c4bdbf1629b6abcd1faa83338cd2fce4747e8390d3290c556
+    patches:
+        - setup.patch
+        - win-c.patch  # [win]
+        - tables_init.patch  # [win]
+
+build:
+    number: 0
+    skip: True  # [win]
```
Comment (scopatz) [inline recipes/pytables/meta.yaml]: This really shouldn't skip windows, though I don't really know anything about windows development.

Comment (ocefpaf) [inline recipes/pytables/meta.yaml]: The history here is to skip Windows, get this in a `feedstock`, iterate there to fix Windows leaving `staged-recipes` CIs free.

However, that is not the main issue with this PR. #1133 is :unamused:

Comment (scopatz) [inline recipes/pytables/meta.yaml]: Ahh understood.

**Comment by ocefpaf** [comment]
@msarahan can you review this one? I am skipping Windows for now but I will tackle it on the feedstock as soon as this is merged.

**Comment by msarahan** [comment]
Thanks @ocefpaf

**Pull request merged**